pkgs/README.md

```diff
@@ -23,7 +23,24 @@ See the [CONTRIBUTING.md](../CONTRIBUTING.md) document for more general informat
 
 ## Quick Start to Adding a Package
 
-To add a package to Nixpkgs:
+We welcome new contributors of new packages to Nixpkgs, arguably the greatest software database known. However, each new package comes with a cost for the maintainers, Continuous Integration, caching servers and users downloading Nixpkgs.
+
+Before adding a new package, please consider the following questions:
+
+* Is the package ready for general use? We don't want to include projects that are going to be abandoned immediately. In case of doubt, check with upstream.
```

Comment:
That being said, a better wording:

```suggestion
* Is the package ready for general use? We don't want to include projects that are too imature or are going to be abandoned immediately. In case of doubt, check with upstream.
```